ncm-pam/src/main/pan/components/pam/config.pan

```diff
@@ -1,25 +1,9 @@
-# ${license-info}
-# ${developer-info}
-# ${author-info}
-# ${build-info}
-
-##########################################################################
-# Coding style: emulate <TAB> characters with 4 spaces, thanks!
-##########################################################################
-
-unique template components/${project.artifactId}/config;
-
-include 'components/pam/schema';
+${componentconfig}
 
 # standard component settings
-"/software/components/pam/version"    = '${no-snapshot-version}';
-"/software/components/pam/active"    ?=  true;
-"/software/components/pam/dispatch"  ?=  true;
 "/software/components/pam/directory" ?= "/etc/pam.d";
```

Comment:
the `{componentconfig}` ends with a relative prefix to the compont path, so you can replace this with relative paths like
```
"directory" ? = ...
```

Reply:
I suppose, it could be a bit fragile if it changes though.